src/helpers.js

```diff
@@ -0,0 +1,32 @@
+import { useRef, useEffect, useState } from 'react';
```

Comment:
What do you think about renaming this file GlobalHelpers.js?
If you like it don't forget to change it in the files this is used.